rest_framework_jwt/views.py

```diff
@@ -106,6 +109,23 @@ class RefreshJSONWebToken(JSONWebTokenAPIView):
     serializer_class = RefreshJSONWebTokenSerializer
 
 
+class DeviceLogout(DestroyAPIView):
+    """
+    Logout user by deleting Device.
+    """
+    queryset = Device.objects.all()
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        return self.queryset.filter(user=self.request.user)
+
+    def get_object(self):
```

Comment:
But the get_object() uses the get_queryset(), but yeah, it can be done in the get_object()